examples/interactivity/pop-ups.html

```diff
@@ -125,52 +125,14 @@
       strokeColor: opacity(BlueViolet, 0.8)
     `);
     const layer = new carto.Layer('layer', source, viz);
-
```

Comment:
Wait, why?
We did this on purpose.

Reply:
Ah, I thought we want unit-examples without extra functionality. This example looks exactly like the interactive-style plus pop-ups. As a developer, I would appreciate small and clean examples.

But, despite this, if you want to add this I could revert the commit.

Reply:
Yup, that's definitely the idea. But since we already introduced the interactivity based styling on the previous example, we thought it wouldn't be a big deal keeping it on the following one. Also, because this is something that a lot of people will want to make, so better giving it to them at once :)

Reply:
OK. It makes sense :+1: